library/init/logic.lean

```diff
@@ -41,8 +41,8 @@ assume hna : ¬a, absurd ha hna
 
 /- false -/
 
-lemma false.elim {c : Prop} (h : false) : c :=
-false.rec c h
+@[inline] def false.elim {C : Sort u} (h : false) : C :=
```

Comment:
You don't need `inline` here, the body of `false.elim` is a "simple" application and should be inlined automatically.